src/EditCocktail/EditCocktail.js

```diff
@@ -0,0 +1,169 @@
+import {
+  useDisclosure,
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalCloseButton,
+  ModalHeader,
+  ModalBody,
+  FormControl,
+  FormLabel,
+  Input,
+  ModalFooter,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import "./EditCocktail.css";
+import Ingredients from "../Ingredients/Ingredients";
+import Steps from "../Steps/Steps";
+
+const EditCocktail = ({ choosenCocktail, updateCocktail, updateSteps }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  const initialRef = React.useRef(null);
+  const finalRef = React.useRef(null);
+
+  const [coctailName, setCocktailName] = useState("");
```

Comment:
It's okay to add comments. For example, you could add `// State` above line 26.

I do like the use of line breaks to separate these lines of code. Don't be afraid to add a few comments to further label the different purposes of each set of lines.